prog/weaveexec/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM alpine
+#FROM alpine
+FROM jeanblanchard/alpine-glibc
```

Comment:
Ah, it looks like the CNI API exec's stuff. Is that correct? If so, how is one supposed to get the required executables (and all their dependencies) into the proxy container?